testConnector/src/main/java/io/vantiq/extsrc/testConnector/TestConnectorHandleConfiguration.java

```diff
@@ -0,0 +1,124 @@
+package io.vantiq.extsrc.testConnector;
+
+import io.vantiq.extjsdk.ExtensionServiceMessage;
+import io.vantiq.extjsdk.Handler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+public class TestConnectorHandleConfiguration extends Handler<ExtensionServiceMessage> {
+    Logger log;
+    String sourceName;
+    TestConnectorCore core;
+
+    boolean configComplete = false; // Not currently used
+
+    // Constants for getting config options
+    private static final String CONFIG = "config";
+    private static final String TEST_CONFIG = "testConfig";
+    private static final String GENERAL = "general";
+    private static final String POLLING_INTERVAL = "pollingInterval";
+
+    // Default pollingInterval if not specified (1 minute in millis)
+    private static final int DEFAULT_POLLING_INTERVAL = 60000;
+
+    public TestConnectorHandleConfiguration(TestConnectorCore core) {
+        this.core = core;
+        this.sourceName = core.getSourceName();
+        log = LoggerFactory.getLogger(this.getClass().getCanonicalName() + "#" + sourceName);
+    }
+
+    @Override
+    public void handleMessage(ExtensionServiceMessage message) {
+        Map<String, Object> configObject = (Map) message.getObject();
+        Map<String, Object> config;
+        Map<String, Object> testConfig;
+        Map<String, Object> general;
+
+        // Obtain entire config from the message object
+        if (!(configObject.get(CONFIG) instanceof Map)) {
+            log.error("Configuration failed. No configuration suitable for Test Connector.");
+            failConfig();
+            return;
+        }
+        config = (Map) configObject.get(CONFIG);
+
+        // Retrieve the jdbcConfig and the vantiq config
```

Comment:
cut/paste -- jdbc....

Reply:
Gone now, sorry about that. Wrapping up the README shortly.